javascript-algorithms-and-data-structures/pokemon-search-app/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Pokémon Search App</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <h1>Pokémon Search App</h1>
+    <div class="container">
+      <form role="search" id="search-form">
+        <label for="search-input" class="sr-only"
+          >Search for Pokémon Name or ID</label
+        >
+        <input
+          type="text"
+          name="pokemon"
+          id="search-input"
+          placeholder="Enter Pokémon Name or ID"
+        />
+        <button id="search-button">Search</button>
+      </form>
+      <div class="output">
+        <div class="top-container">
+          <div class="name-and-id">
+            <span id="pokemon-name"></span>
+            <span id="pokemon-id"></span>
+          </div>
+          <div class="size">
+            <span id="weight"></span>
+            <span id="height"></span>
+          </div>
+          <img
+            src="./images/search.png"
+            alt="default Pikachu search sprite"
+            id="default-sprite"
+          />
+          <div id="types"></div>
+        </div>
+        <div class="bottom-container">
+          <table>
+            <tr>
+              <th>Base</th>
+              <th>Stats</th>
+            </tr>
+            <tr>
+              <td>HP:</td>
+              <td id="hp"></td>
+            </tr>
+            <tr>
+              <td>Attack:</td>
+              <td id="attack"></td>
+            </tr>
+            <tr>
+              <td>Defense:</td>
+              <td id="defense"></td>
+            </tr>
+            <tr>
+              <td>Sp. Attack:</td>
+              <td id="special-attack"></td>
+            </tr>
+            <tr>
+              <td>Sp. Defense:</td>
+              <td id="special-defense"></td>
+            </tr>
+            <tr>
+              <td>Speed:</td>
+              <td id="speed" class="speed"></td>
+            </tr>
+          </table>
+        </div>
+      </div>
+    </div>
+    <script src="./script.js"></script>
```

Comment:
since the javascript file name was changed here we should also change the file name itself.
Right now, it still reads as index.js

Reply:
My bad, forgot to update this. I have fixed it.